src/child/actions/window.js

```diff
@@ -57,30 +57,104 @@ export function resizeError() {
     };
 }
 
-export function resizeToCompact() {
+export function updatingOptions() {
```

Comment:
I don't think `updatingOptions()`, `updatingOptionsSuccess()` or `updatingOptionsError()` should be exported. (Could potentially do this if they were being tested directly, but as they're not, lets not expose them.)